cuda_bindings/docs/source/install.md

```diff
@@ -27,12 +27,12 @@ Install all optional dependencies with:
 pip install -U cuda-python[all]
 ```
 
-Where the optional dependencies are:
+Where the optional dependencies are now managed through the ``cuda-toolkit`` metapackage for improved version constraints and dependency resolution:
 
-* nvidia-cuda-nvrtc (Provides NVRTC shared library)
-* nvidia-nvjitlink (Provides nvJitLink shared library)
-* nvidia-cuda-nvcc (Provides NVVM shared library)
-* nvidia-cufile (Provides cuFile shared library)
+* ``cuda-toolkit[nvcc,nvrtc,nvjitlink,nvvm]==13.*`` (Provides NVRTC, nvJitLink, and NVVM shared libraries)
+* ``cuda-toolkit[cufile]==13.*; sys_platform == 'linux'`` (Provides cuFile shared library on Linux)
+
+The ``cuda-toolkit`` metapackage behaves similarly to conda's ``cuda-version``, providing more predictable version constraints and safer dependency resolution compared to individual NVIDIA wheel packages.
```

Comment:
See my earlier comment (https://github.com/NVIDIA/cuda-python/pull/883#issuecomment-3212579853). I want to downplay the comparison against conda's `cuda-version` because `cuda-toolkit` still does not fully live up to my expectation. Let's just follow my other comment above and mention optional dependencies are installed via `cuda-toolkit`.